src/patients/GeneralInformation.tsx

```diff
@@ -9,25 +9,28 @@ import TextInputWithLabelFormGroup from '../components/input/TextInputWithLabelF
 import SelectWithLabelFormGroup from '../components/input/SelectWithLableFormGroup'
 import DatePickerWithLabelFormGroup from '../components/input/DatePickerWithLabelFormGroup'
 
+interface Feedback {
+  givenName: string
+  dateOfBirth: string
+}
+
+interface Invalids {
+  givenName: boolean
+  dateOfBirth: boolean
+}
+
 interface Props {
   patient: Patient
   isEditable?: boolean
   errorMessage?: string
   onFieldChange?: (key: string, value: string | boolean) => void
-  isInvalid?: boolean
-  patientGivenNameFeedback?: string
+  invalids?: Invalids
```

Comment:
Can we use a better name for this prop?

Reply:
Sure, I agree is not even close to be good. My bad!